Whitelist-dApp.md

```diff
@@ -234,7 +234,7 @@ npm install web3modal
 In the same terminal also install `ethers.js`
 
 ```bash
-npm install ethers
+npm install ethers@5
```

Comment:
Thanks. This might be the case with all the other lessons as well, unfortunately. 

Could you add a note below explaining why? 

Ex:

> Note : We install v5 specifically since the new v6 has breaking changes to the code.

Reply:
<img width="454" alt="image" src="https://user-images.githubusercontent.com/18665326/216665184-7eb9508b-074e-4197-9cad-e4ac2d4e810a.png">

looks like there are 3 places in code

I can quick test this installs in every project tomorrow, and will update all

Reply:
should I add note in every place?

Reply:
@yanukadeneth99

Reply:
Please do @happylolonly, thanks!

Reply:
@happylolonly make sure you pull from main. There are few updates that were added.

Reply:
hey, couldn't check at Sat, will do it next hours

Reply:
Cool, thanks.